client/my-sites/scan/main.tsx

```diff
@@ -231,40 +233,29 @@ class ScanPage extends Component< Props > {
 		return this.renderScanOkay();
 	}
 
-	maybeRenderHistory() {
-		const { filter } = this.props;
-
-		const isJetpackCom = !! config( 'env_id' ).includes( 'jetpack-cloud' );
-
-		return (
-			! isJetpackCom && (
-				<div className="scan__history">
-					{ this.renderHeader( 'History' ) }
-					<ThreatHistoryList { ...{ filter } } />
-				</div>
-			)
-		);
-	}
-
 	render() {
 		const { siteId } = this.props;
+		const isJetpackPlatform = isJetpackCloud();
+
 		if ( ! siteId ) {
 			return;
 		}
-		const isJetpackCom = !! config( 'env_id' ).includes( 'jetpack-cloud' );
+
 		return (
 			<Main
```

Comment:
To match Backup and some other Calypso pages, add `wideLayout={ ! isJetpackCloud() }`. I'm not sure this is the logic we'll stick with long-term, but for now it appears to work.

Reply:
Thks! Updated.